### ISSUE TITLE
`tput init` makes the cursor jump to start

### ISSUE THREAD

**Issue by diagramatics** [zeit/hyper#3537]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS 10.14.3
- **Hyper.app version**: 3.0.0-canary.8
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/diagramatics/b9d3512eded6f9f7d6e9274e1f652dbe
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**:  Yes

## Issue

So I don't know whether to report this upstream to `xterm.js` since `vscode`'s terminal behaves the same too, so I tried here first. Sorry if this is actually upstream 🙇‍♀️ 

![Kapture 2019-03-25 at 11 57 56](https://user-images.githubusercontent.com/5153378/54888880-4bf7dd00-4ef5-11e9-9b08-bed8cea9c922.gif)

Basically when you call `tput init` the cursor jumps to the start of the terminal again.

This doesn't happen on normal `Terminal.app`.

![Kapture 2019-03-25 at 12 00 18](https://user-images.githubusercontent.com/5153378/54888922-95482c80-4ef5-11e9-9b49-2eb8e05ab7d2.gif)
